instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/jvm/JvmMetrics.scala

```diff
@@ -168,6 +185,22 @@ object JvmMetrics {
           register(MemoryPoolMax, region)
         )
       })
+
+    def bufferPoolInstruments(poolName: String): BufferPoolInstruments = {
+      _memoryBuffersCache.getOrElseUpdate(poolName, {
+        val commonTags = tags
+          .withTag("pool", poolName)
+        val usageTags = commonTags.withTag("measure", "used")
+        val capacityTags = commonTags.withTag("measure", "capacity")
```

Comment:
Do not include these tags. The `used` or `capacity` details are already defined in the metric name.